examples/md-flexible/tests/ThermostatTest.cpp

```diff
@@ -111,13 +117,45 @@ TEST_F(ThermostatTest, MultiComponentTest) {
     EXPECT_NEAR(temperature, targetTemperature1, 0.1);
   }
 
+  // Check no particles have zero temperature (indicating brownian motion hasn't been applied).
+  // We also want to later compare against the current velocity. For simplicity's sake, we use the oldF buffer of the
+  // particle to do so.
+  for (auto iter = _autopas.begin(); iter.isValid(); ++iter) {
+    for (size_t dim = 0; dim < 3; ++dim) {
+      // Check some velocity applied.
+      EXPECT_THAT(iter->getV()[dim], ::testing::Not(::testing::DoubleNear(0, 1e-12)));
+    }
+    // set the oldF to the velocity
+    iter->setOldF(iter->getV());
+  }
```

Comment:
```suggestion
  for (auto &particle : _autopas) {
    for (size_t dim = 0; dim < 3; ++dim) {
      // Check some velocity applied.
      EXPECT_THAT(particle.getV()[dim], ::testing::Not(::testing::DoubleNear(0, 1e-12)));
    }
    // set the oldF to the velocity
    particle.setOldF(particle.getV());
  }
```
I think range-based for loops are a bit nicer to read.